src/models/thelio-spark-b1-n2/repairs.md

```diff
@@ -0,0 +1,630 @@
+
+![CPU (removed)](./img/cpu-removed.webp)
+
+7. Gently place the new CPU into the socket.
+    - The triangle on the CPU should be oriented to match the triangle on the CPU cover, pointing towards the bottom left of the motherboard.
+8. Flip the CPU cover back onto the CPU and push the locking pin down into place.
+9. Reinstall the cooler mounting bracket by lining it up with the holes in the motherboard and pushing the plastic retention pins back into place.
+    - Ensure that the transparent sleeves in the corners of the mounting bracket are pulled outwards to the `1700` position.
+    - The orientation of the cooler mounting bracket doesn't matter.
+
+![CPU cooler mounting bracket corner sleeves](./img/cpu-cooler-mounting-bracket-corner-sleeves.webp)
+
+### Steps to install the thermal paste/CPU cooler:
+
+1. Draw an `X` shape of thermal paste onto the CPU.
+
+![Thermal paste application](./img/thermal-paste-application.webp)
+
+2. Place the CPU cooler onto the CPU; while holding it in place, tighten the screws on the top and bottom of the vertical mounting bar to tighten the 90-degree clips onto the mounting bracket.
+    - The cooler should be oriented so the fan is on the right side (towards the front of the case).
+    - The 90-degree clips must be oriented inward to fasten onto the mounting bracket.
+
+![CPU cooler mounting bracket & 90-degree clips](./img/cpu-cooler-mounting-clips.webp)
+
+3. Set the computer upright, then reinstall the cooler-mounted CPU fan (if necessary), RAM, CPU duct, GPU, side bracket, and top case.
+
+## Replacing the power supply:
+
+The power supply unit (PSU) can be replaced with another unit of the same model. One possible preinstalled model is the [Thermaltake Toughpower GX2 600W (80+ Gold)](https://thermaltakeusa.com/products/toughpower-gx2-600w-ps-tpd-0600nnfagu-2#tab-label-specification).
+
+The PSU is not modular, so all of its cabling will need to be unplugged and removed, then the new PSU's cabling will need to be run through the system and connected.
+
+**Tools required:** Cross-head (Phillips) screwdriver  
+**Time estimate:** 45 minutes  
+**Difficulty:** Medium <span style="color:orange;">●</span>
+
+### Steps to replace the power supply:
+
+1. Follow the steps above to [remove the top case](#removing-the-top-case), [remove the side brace](#removing-the-side-brace), and [unplug the GPU](#replacing-the-gpu).
+2. Locate and unplug all of the cables emanating from the power supply (shown below), including the following:
+
+![PSU cabling source](./img/psu-cables-source.webp)
+
+- The 8-pin CPU power cable labeled `ATX12V1` near the top-left of the motherboard, highlighted green below.
+- The 24-pin motherboard power cable labeled `ATXPWR1` on the right side of the motherboard, highlighted yellow below.
+
+![PSU cabling to the motherboard & CPU](./img/psu-cables-motherboard.webp)
+
+- The 4-pin molex connector labeled `POWER1` on the Thelio Io board.
```

Comment:
Should we refer to it as a [Berg connector](https://en.wikipedia.org/wiki/Berg_connector) rather than Molex? If so, this will have to be updated in multiple places, including on the SATA backplane section.

Reply:
Yes, I got that word mixed up.